### PR TITLE
.gitignore does not commit dist folder and removed dist folder from repo

### PR DESCRIPTION
**Pull request by erikmjohnson**
.gitignore does not commit dist folder and removed dist folder from repo

### CODE REVIEW

File: stories/google-logout.stories.js [hunk 1]
```diff
@@ -37,6 +37,6 @@ story.add('Default Button', () => (
     </GoogleLogout>
   ),
   {
-    jest: ['google-logout.test.js', 'google-logout-dist.test.js'],
+    jest: ['google-logout.test.js', 'google-logout-dist.nottest.js'],
```
Comment (anthonyjgrove) [inline stories/google-logout.stories.js]: you can remove `, 'google-logout-dist.nottest.js'`

File: stories/google-login.stories.js [hunk 2]
```diff
@@ -37,6 +37,6 @@ story.add('Default Button', () => (
   </GoogleLogin>
 ),
   {
-    jest: ['google-login.test.js', 'google-login-dist.test.js'],
+    jest: ['google-login.test.js', 'google-login-dist.nottest.js'],
```
Comment (anthonyjgrove) [inline stories/google-login.stories.js]: remove , 'google-logout-dist.nottest.js'

File: __test__/google-login-dist.nottest.js [hunk 3]
```diff
@@ -0,0 +1,189 @@
+// import React from 'react'
```
Comment (anthonyjgrove) [inline __test__/google-login-dist.nottest.js]: lets just delete these instead of renaming them

File: .gitignore [hunk 4]
```diff
@@ -8,3 +8,4 @@ yarn.lock
 .vscode
 .jest-test-results.json
 docs/
+!dist/
```
Comment (anthonyjgrove) [inline .gitignore]: why do you have the `!`

File: stories/google-logout.stories.js (hunk 1 above)
Comment (erikmjohnson) [reply stories/google-logout.stories.js]: Removed

File: stories/google-login.stories.js (hunk 2 above)
Comment (erikmjohnson) [reply stories/google-login.stories.js]: removed

File: .gitignore (hunk 4 above)
Comment (erikmjohnson) [reply .gitignore]: removed

File: __test__/google-login-dist.nottest.js (hunk 3 above)
Comment (erikmjohnson) [reply __test__/google-login-dist.nottest.js]: removed file

**Pull request merged**